MonitorControl/Support/Utils.swift

```diff
@@ -242,4 +119,196 @@ class Utils: NSObject {
     /// Don't listen for any keys
     case none = 3
   }
+
+  enum PollingMode {
+    case none
+    case minimal
+    case normal
+    case heavy
+    case custom(value: Int)
+
+    var value: Int {
+      switch self {
+      case .none:
+        return 0
+      case .minimal:
+        return 5
+      case .normal:
+        return 10
+      case .heavy:
+        return 100
+      case let .custom(val):
+        return val
+      }
+    }
+  }
+
+  public enum Command: UInt8 {
```

Comment:
Nitpicking here, but I would move these to a separate file and keep the DDC naming for it probably, makes it a bit more explicit, but it's up to you.

Reply:
I was unsure about this. In an earlier commit this was in a separate place but since other enums are located in Utils (it has its own section) so that's why I settled with this. The Command enum sometimes used in a non-DDC context as well (since software control and DisplayServices control was introduced) so that's why I did not use the DDC naming. But there are equally valid reasons to think the opposite way. :)

What I now did is that I moved it to a separate Command.swift but did not add the DDC naming. Hope you agree with this.